src/raftstore/store/peer.rs

```diff
@@ -207,6 +208,7 @@ pub struct Peer {
     pub delete_keys_hint: u64,
     /// approximate region size.
     pub approximate_size: Option<u64>,
```

Comment:
why not combine approximate_size and approximate_size_expired_time together, they have the same validity time.